### PR TITLE
[ERC] MetaProxy Factory Standard

### PR DESCRIPTION
**Pull request by pinkiebell**
First draft of the MetaProxy Factory (ERC)

Reference  Implementation: https://github.com/pinkiebell/MetaProxy

### CODE REVIEW

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-3448.md is new and new files must be reviewed

**Comment by pinkiebell** [comment]
👋  @MicahZoltu,

any chance to get this merged as a draft?

**Comment by MicahZoltu** [comment]
@lightclient handles ERCs now.

File: EIPS/eip-3448.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+## Test Cases
+Test cases include:
+- invocation with no arguments
+- invocation with arguments
+- invocation with return values
+- invocation with revert (confirming reverted payload is transferred)
+
+Tests for these cases are included in the reference implementation project.
+
+## Reference Implementation
```
Comment (lightclient) [inline EIPS/eip-3448.md:62]:
```suggestion
## Implementation
```

**Review by lightclient — APPROVED** [review]

**Pull request merged**